### ISSUE TITLE
Display Whitelist Timelock Transactions

### ISSUE THREAD

**Issue by gakonst** [loomnetwork/dashboard#95]
Each validator has deployed a timelock contract which holds the funds they have locked for 1 year.

After validators deployed the contract, they get whitelisted with an amount equal to their timelock. 

When displaying granular information about validators, we want to show their timelock contract etherscan transactions, so as to justify their whitelist amounts.

In addition, we also want to display the information about their delegations. We can do that by getting the output of `getDelegationsAsync(candidate)` and putting these in an a table.

So basically we want 2 headers:
- 1 for timelocked funds, which is going to be a couple of rows which show the transactions the validators did to justify their whitelist amounts

- 1 header which has a list (maybe table with 50 rows per page e.g.) of all delegations for that validator, which will be the output of `dpos.getdelegations(validator)` (edited) 

![image](https://user-images.githubusercontent.com/17802178/53231560-e4e6ce80-3691-11e9-8d0b-f796e998d357.png)

**Comment by redben** [comment]
@gakonst do we still want this?

**Comment by gakonst** [comment]
We do that already, it's the self-delegated stake. For greater transparency we might want to hardcode etherscan links to the lock transactions.
